client/landing/jetpack-cloud/components/scan-threats/index.tsx

```diff
@@ -51,8 +51,7 @@ const ScanError: React.FC< { site: Site } > = ( { site } ) => {
 
 	return (
 		<div className="scan-threats__error">
-			{ translate( 'The scanner was unable to check all files and errored before completion.' ) }
-			<br />
+			{ translate( 'The scanner was unable to check all files and errored before completion. ' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.